runtime/near-vm-logic/src/dependencies.rs

```diff
@@ -276,6 +278,49 @@ pub trait External {
         prepaid_gas: Gas,
     ) -> Result<()>;
 
+    /// Attach the [`FunctionCallAction`] action to an existing receipt.
+    ///
```

Comment:
Assumed the docs for this would be lighter since it's documented in `promise_batch_action_function_call_weight`, but I see the benefit in giving some info here also.

Reply:
it is fine to have lighter docs - but then let's make sure that we have a 'pointer' to the place where people can read more details.

(I actually don't know which file is read by users - this one or logic.rs?)